app/Http/Kernel.php

```diff
@@ -49,6 +49,7 @@ class Kernel extends HttpKernel
         'api' => [
             //Do this in the routes 'throttle:rate_limit,1',
             'bindings',
+            'api.log',
```

Comment:
You can just write `\App\Http\Middleware\ApiLogMiddleware::class` instead of `'api.log'` and remove it later